docs/get_started/quickstart.mdx

```diff
@@ -532,6 +532,136 @@ go build .
 
 </details>
 
+</TabItem>
+<TabItem value="python" label="Python" default>
+
+        <Admonition type="note" title="Prerequisites">
+            - Python >= v3.11
+            - Optional but recommended: [Install the Restate Server & CLI](/develop/local_dev#running-restate-server--cli-locally)
+        </Admonition>
+
+        <Step stepLabel="1" title="Get the Greeter service template">
+
+            <CH.Code>
+                ```shell CLI
+                # link(1:3) /develop/local_dev#running-restate-server--cli-locally
+                restate example python-hello-world &&
+                cd python-hello-world
+                ```
+
+                ```shell npx
+                npx @restatedev/restate example python-hello-world &&
+                cd python-hello-world
+                ```
+
+                ```shell wget
+                wget https://github.com/restatedev/examples/releases/latest/download/python-hello-world.zip &&
+                unzip python-hello-world.zip -d python-hello-world &&
+                rm python-hello-world.zip && cd python-hello-world
+                ```
+
+            </CH.Code>
+
+        </Step>
+        <Step stepLabel="2" title="Run the Greeter service">
+
+            Create a `venv` and install the requirements:
+
+            ```shell
+            python3 -m venv .venv
+            source .venv/bin/activate
+            pip install -r requirements.txt
+            ```
+
+            Now, start developing your service in `example.py`. Run it with a [Hypercorn](https://pypi.org/project/Hypercorn/) server, and let it listen on port `9080` for requests:
+
+            ```shell
+            python -m hypercorn --config hypercorn-config.toml -b localhost:9080 example:app
```

Comment:
I don't think you need to specify the bind address here, it's in the config: https://github.com/restatedev/examples/blob/main/templates/python/hypercorn-config.toml